alns/tests/test_alns.py

```diff
@@ -134,6 +163,89 @@ def repair_operator():  # placeholder
     assert_(operator is repair_operator)
 
 
+def test_compute_op_coupling():
+    """
+    Tests if the compute_op_coupling method correctly computes the matrix
+    that describes the dependencies between repair and destroy operators.
+    """
+    alns = ALNS()
+
+    d_operators = get_destroy_operators(2)
+    r_operators = get_repair_operators(2)
+
+    for d_op in d_operators:
+        alns.add_destroy_operator(d_op)
+
+    for r_op in r_operators:
+        alns.add_repair_operator(r_op)
+
+    op_coupling = alns._compute_op_coupling()
+
+    assert (op_coupling == np.array([[1, 1], [1, 1]])).all()
```

Comment:
simplify: `assert_almost_equal(op_coupling, np.ones((2, 2))`

Built-in `assert` has the additional problem that they are omitted if you run python with the `-O` flag, in which case all such tests always pass. That is why I avoid them in test cases.